templates/vue/src/store/getters.js

```diff
@@ -87,7 +87,10 @@ export function isVisible(state, { getNode, hasMultiContentAncestor }) {
       return false
     }
     if (!Helpers.hasPermission(node, "edit", showRejected)) {
-      return !hasMultiContentAncestor(node.id)
+      return (
+        (node.accessible || !node.hideWhenLocked) &&
```

Comment:
Similarly, would it be better to use `page.unlocked` here? I am not sure of the exact meaning of `accessible` vs `unlocked`, but this would increase consistency.